app/config/config.yml

```diff
@@ -32,7 +32,7 @@ framework:
     session:
         handler_id:  ~
     fragments:       ~
-    http_method_override: true
+    http_method_override: false
```

Comment:
can you explain why do you need to change this value?

Reply:
nope, it doesn't need to be changed. Reverted.